### ISSUE TITLE
False positive in case of identity schema

### ISSUE THREAD

**Issue by remyleone** [bflad/tfproviderlint#340]
### tfproviderlint and terraform-plugin-sdk Version

```console
❯ go tool tfproviderlint --version
tfproviderlint v0.15.0-dev

$ go mod graph | cut -d' ' -f 1 | grep terraform-plugin-sdk | uniq
github.com/hashicorp/terraform-plugin-sdk/v2@v2.37.0
```

### Affected Check(s) or Function(s)

* S013

### Expected Behavior

I should not have a linter that tell me that I should configure one of Computed, Optional, or Required in a identity schema.

```go
		Identity: &schema.ResourceIdentity{
			Version: 0,
			SchemaFunc: func() map[string]*schema.Schema {
				return map[string]*schema.Schema{
					"volume_id": {
						Type:              schema.TypeString,
						RequiredForImport: true,
						Description:       "Volume ID",
					},
					"zone": {
						Type:              schema.TypeString,
						RequiredForImport: true,
						Description:       "Zone",
					},
				}
			},
		},
```

### Actual Behavior

I got the following error happening:
```
S013: schema should configure one of Computed, Optional, or Required
```

In case I have a `Computed: true` defined in this schema, then I got an internal error.
